src/Servers/IIS/AspNetCoreModuleV2/CommonLibTests/NativeTests.targets

```diff
@@ -1,8 +0,0 @@
-<Project>
-  <PropertyGroup>
-    <VsTestConsole>$(VCIDEInstallDir)..\CommonExtensions\Microsoft\TestWindow\vstest.console.exe</VsTestConsole>
```

Comment:
Does this need to move into Cpp.Common.targets?

Reply:
It wasn't being used. At least we don't think it was.